.changelog/unreleased/bug-fixes/195-revoting-bug-fix.md

```diff
@@ -0,0 +1,2 @@
+- Revoting bug fix.
```

Comment:
Fix a bug that would prevent users from switching their vote under specific circumstances.